projects/packages/sync/src/replicastore/class-table-checksum.php

```diff
@@ -323,6 +351,7 @@ private function prepare_filter_values_as_sql( $filter_values = array(), $table_
 			switch ( $filter['operator'] ) {
 				case 'IN':
 				case 'NOT IN':
+				case '<>':
```

Comment:
This won't work as expected.

Adding multiple arguments to a `<>` with this syntax will create the following query:

`SELECT * FROM wp_postmeta WHERE meta_key <> ('_wp_page_template', '_edit_lock')`

that will result in `ERROR 1241 (21000) at line 1: Operand should contain 1 column(s)`.

The proper syntax generated by `<>` should be:

```
column <> value AND column <> value2 AND column <> value3 
```

Reply:
:thumbsup: The <> was how the existing sql statement was as we were looking at a single status of 'spam'. No harm in switching to use NOT IN.